corehq/apps/api/odata/serializers.py

```diff
@@ -137,7 +137,7 @@ def _lookup(item, xform_json):
 
             for i, xform_json in enumerate(data['value']):
                 data['value'][i] = {
-                    item.label.split('.')[-1].replace('#', '').replace('@', ''): _lookup(item, xform_json)
+                    get_odata_property_from_export_item(item): _lookup(item, xform_json)
```

Comment:
way ahead of me!

Reply:
(still would be good to have a few unit tests that also serve to document this method)

Reply:
966354af720e49993a3481971da63d92f0d83eda